resource-management/pkg/store/redis/redis.go

```diff
@@ -177,6 +177,50 @@ func (gr *Goredis) PersistVirtualNodesAssignments(virtualNodeAssignment *store.V
 	return true
 }
 
+// Get batch node ids for node status inquiry
+func (gr *Goredis) BatchLogicalNodesInquiry(batchSize uint64) []*types.LogicalNode {
```

Comment:
How is batchSize used? Why it needs to be declared as uint64 and convert to int64 immediately before it was used?

Reply:
BatchSize should be "int64" type as gr.client.Scan required.  batchSize is changed to "init64".